lib/web/ui/integration.go

```diff
@@ -204,6 +204,27 @@ type AWSOIDCListEC2Response struct {
 	NextToken string `json:"nextToken,omitempty"`
 }
 
+// AWSOIDCListSecurityGroupsRequest is a request to ListSecurityGroups using the AWS OIDC Integration.
+type AWSOIDCListSecurityGroupsRequest struct {
+	// Region is the AWS Region.
+	Region string `json:"region"`
+	// VPCID is the VPC to filter EC2 Instance Connect Endpoints.
+	VPCID string `json:"vpcId"`
+	// NextToken is the token to be used to fetch the next page.
+	// If empty, the first page is fetched.
+	NextToken string `json:"nextToken"`
+}
+
+// AWSOIDCListSecurityGroupsResponse contains a list of VPC Security Groups and a next token if more pages are available.
+type AWSOIDCListSecurityGroupsResponse struct {
+	// SecurityGroups contains the page of SecurityGroups
+	SecurityGroupss []awsoidc.SecurityGroup `json:"securityGroups"`
```

Comment:
Extra `s` at the end.
```suggestion
	SecurityGroups []awsoidc.SecurityGroup `json:"securityGroups"`
```